src/Ombi/ClientApp/src/app/requests-list/components/movies-grid/movies-grid.component.ts

```diff
@@ -102,7 +103,15 @@ export class MoviesGridComponent implements OnInit, AfterViewInit {
                     // Flip flag to show that loading has finished.
                     this.isLoadingResults = false;
                     this.resultsLength = data.total;
-
+                    if (data.collection.filter(x => x.requestedUserId != null).length > 0 &&
+                      data.collection.filter(x => x.requestedUser != null).length > 0) {
+                      this.anonimized = false;                      
+                    }
+                    if (this.anonimized) {
+                      this.displayedColumns.forEach((element, index) => {
+                        if (element == 'requestedUser.requestedBy') this.displayedColumns.splice(index, 1);
```

Comment:
```suggestion
                        if (element === 'requestedUser.requestedBy')
                        {
                            this.displayedColumns.splice(index, 1);
                        }
```

Reply:
JS/TS not the main language i code in and i always mess up strict equality :')